package.json

```diff
@@ -45,8 +39,28 @@
     "graphql": "^0.13.2",
     "graphql-tag": "^2.9.2",
     "graphql_ppx": "^0.2.4",
+    "husky": "^1.2.0",
+    "lint-staged": "^8.1.0",
     "react-apollo": "^2.1.6",
     "reason-react": "^0.5.3"
   },
-  "dependencies": {}
+  "dependencies": {},
+  "lint-staged": {
+    "*.re": [
+      "refmt --in-place",
```

Comment:
you should use bsrefmt instead because it would work if bs-platform is installed

Reply:
what's the diff between refmt and bs-refmt?

Reply:
bsrefmt is just refmt but it's bundled with bs-platform,